### PR TITLE
286 Enhance brats tutorial

### LINKED ISSUE #286: Update Brats tutorial
The [brats tutorial](https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/brats_segmentation_3d.ipynb) can be enhanced for the following places:
1. `CenterSpatialCropd` should be replaced by `sliding_window_inference`. Otherwise, the calculated score will always exclude the cropped boundaries. 
2. `RandFlipd` should be used for all spatial dimensions for better performance.

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Signed-off-by: Yiheng Wang <vennw@nvidia.com>

Fixes #286 .

### Description
Except for the enhancements mentioned in #286 , this PR also adds evaluation on original image spacings.

### Status
**Ready**

### Checks

- [x] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Comment by yiheng-wang-nv** [comment]
Hi @dongyang0122 , can you please also review this PR? It also adds the `Invertd` transform to do evaluation on original image spacings.

**Comment by Nic-Ma** [comment]
Thanks for the update.
Please delete the training log.

File: 3d_segmentation/brats_segmentation_3d.ipynb [hunk 1]
```diff
@@ -47,7 +47,7 @@
   },
```
Comment (Nic-Ma) [inline 3d_segmentation/brats_segmentation_3d.ipynb:47]: <pre class="ql-syntax" spellcheck="false">Line #6.        download=False,
</pre><p> </p><p>Please don't change to False.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Review by Nic-Ma — APPROVED** [review]
Looks good to me except for minor comments.

Thanks.

**Pull request merged**